cmd/node/config/config.go

```diff
@@ -0,0 +1,55 @@
+// MIT License. Copyright (c) 2020 CQFN
+// https://github.com/cqfn/degitx/blob/master/LICENSE
+
+// Package config contains yaml node config parser
+package config
```

Comment:
@stepanvalyavskiy I think we can put `config.go` near `main.go` at `./cmd/node` directory, since it's not supposed to be used outside of CLI app, so we can reduce the scope to one package